examples/nlp_finetuning/renate_config.py

```diff
@@ -0,0 +1,52 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+from pathlib import Path
+from typing import Callable, Optional, Union
+
+import torch
+import transformers
+
+import renate.defaults as defaults
+from renate.benchmark.datasets.nlp_datasets import HuggingfaceTextDataModule
+from renate.benchmark.scenarios import Scenario
+from renate.models import RenateModule
+from renate.models.renate_module import RenateWrapper
+
+
+def model_fn(model_state_url: Optional[Union[Path, str]] = None) -> RenateModule:
+    """Returns a DistilBert classification model."""
+    transformer_model = transformers.DistilBertForSequenceClassification.from_pretrained(
+        "distilbert-base-uncased", num_labels=2, return_dict=False
+    )
+    model = RenateWrapper(transformer_model, loss_fn=torch.nn.CrossEntropyLoss())
+    if model_state_url is not None:
+        state_dict = torch.load(str(model_state_url))
+        model.load_state_dict(state_dict)
+    return model
+
+
+def data_module_fn(
+    data_path: Union[Path, str], chunk_id: int, seed: int = defaults.SEED
```

Comment:
replace `chunk_id: int` with `dataset_name: str` and enjoy the magic :)

Reply:
Right, forgot about that.